electrum/gui/kivy/uix/dialogs/cpfp_dialog.py

```diff
@@ -0,0 +1,161 @@
+from typing import TYPE_CHECKING, Optional
+
+from kivy.app import App
+from kivy.factory import Factory
+from kivy.properties import ObjectProperty
+from kivy.lang import Builder
+
+from electrum.gui.kivy.i18n import _
+
+if TYPE_CHECKING:
+    from ...main_window import ElectrumWindow
+
+from .fee_dialog import FeeSliderDialog
+from electrum.gui.qt.amountedit import BTCAmountEdit
+from electrum.util import format_satoshis_plain
+
+Builder.load_string('''
+<CPFPDialog@Popup>
+    title: _('CPFP')
```

Comment:
Why not write out the full name? There is enough space for it AFAICT.

```suggestion
    title: _('Child Pays for Parent')
```